lib/container.d.ts

```diff
@@ -12,21 +14,21 @@ declare class Container {
      * @param  {Modem}  modem Modem to connect to the remote service
      * @param  {string} id    Container id (optional)
      */
-    constructor(modem: any, id?: any);
+    constructor(modem: Modem, id?: string);
     /**
      * Get the list of containers
      * https://docs.docker.com/engine/reference/api/docker_remote_api_v1.24/#/list-containers
      * @param  {Object}   opts  Query params in the request (optional)
      * @return {Promise}        Promise returning the result as a list of containers
      */
-    list(opts: any): Promise<{}>;
+    list(opts?: any): Promise<{}>;
```

Comment:
yes, sure, {} is a type here, but here it should be :
`async list(opts?: any): Container[] {`